docs/phdata.rst

```diff
@@ -93,9 +95,9 @@ Measurement specs
 ^^^^^^^^^^^^^^^^^
 
 The optional **/photon_data/measurement_specs** group contains additional
-information that allows to unambiguously interpret the data for each specific
-type of measurement. This group is optional, but if present it must be
-complete.
+information allowing unambiguous interpretration of the data for each specific
+type of measurement. This group must be
+complete if present (**Should it be discarded if incomplete?**).
```

Comment:
You can always scrape whatever information you find in the file. My point is, either we save RAW timestamps without metadata, or a minimal set of `measurement_specs` field sufficient for the complete interpretation. If you provide some of those fields the file is "broken" or equivalent to a RAW one.